types/p-queue/index.d.ts

```diff
@@ -1,4 +1,4 @@
-// Type definitions for p-queue 2.3
+// Type definitions for p-queue 3.0
```

Comment:
When upgrading the major version, please follow the steps outlined here: https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version